rust/.env.example

```
@@ -1,4 +1,8 @@
 MIFOSX_BASE_URL=https://localhost:8443/fineract-provider/api/v1
 MIFOSX_TENANT_ID=default
-MIFOSX_USERNAME=mifos
-MIFOSX_PASSWORD=password
+MIFOSX_USERNAME=<your_username>
+MIFOSX_PASSWORD=<your_password>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MIFOSX_PASSWORD key should go before the MIFOSX_TENANT_ID key

(UnorderedKey)